app/controllers/spree/api/v2/storefront/feedback_reviews_controller.rb

```diff
@@ -4,7 +4,7 @@ module V2
       module Storefront
         class FeedbackReviewsController < ::Spree::Api::V2::ResourceController
           before_action :load_review, only: :create
-          before_action :require_spree_current_user if Spree::Reviews::Config[:require_login]
+          before_action :require_spree_current_user
```

Comment:
this change does not make sense..

we should make this work as configurable..instead of forcing the behavior